src/nrepl/connectSequence.ts

```diff
@@ -335,24 +345,62 @@ function getDefaultCljsType(cljsType: string): CljsTypeConfig {
   return defaultCljsTypes[cljsType];
 }
 
+function getDefaultSequence(
```

Comment:
Considering my other comment about using the term "default", maybe we can change this to something like `getUserSpecifiedSequence`.